prow/plugins/label/label.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	"github.com/sirupsen/logrus"
 
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
nit: could we group this with third-party dependencies instead of with `test-infra` ones?

Reply:
Happy to follow the proposed change. Where do we group test-infra deps?